.github/workflows/tests.yml

```diff
@@ -531,6 +536,11 @@ jobs:
       matrix:
         os: [ ubuntu-latest, macos-13, macos-latest, windows-latest ]
         fips: [ 0, 1 ]
+        exclude:
+          # macOS aarch64 fix for FIPS pending the following PR applied to FIPS branch:
+          # https://github.com/aws/aws-lc/pull/2005
+          - os: 'macos-latest'
+            fips: 1
```

Comment:
can we just go ahead and apply PR 2005 to the FIPS branch instead of excluding it here?

Reply:
That PR fix didn't get merged before 1.59.0.  I'll take note to remove this the next time we re-align aws-lc-sys.